packages/terra-consumer-nav/src/Nav.jsx

```diff
@@ -1,31 +1,126 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import classNames from 'classnames/bind';
-import 'terra-base/lib/baseStyles';
-import styles from './Nav.scss';
-
-const cx = classNames.bind(styles);
+import QuickLinks from './components/quick-links/QuickLinks';
+import NavItems from './components/nav-items/NavItems';
+import './Nav.scss';
```

Comment:
You'll want to use CSS modules.. it helps scope the CSS. See [badge](https://github.com/cerner/terra-core/blob/master/packages/terra-badge/src/Badge.jsx#L5) as an example